spec/features/tagging_during_migration_spec.rb

```diff
@@ -0,0 +1,85 @@
+require "rails_helper"
+
+RSpec.describe "Tagging content during migration", type: :feature do
+  include PublishingApiHelper
+
+  before do
+    given_we_can_populate_the_dropdowns_with_content_from_publishing_api
+  end
+
+  scenario "User makes a change to a item without all the link types" do
+    given_there_is_an_item_that_can_have_only_one_link_type
+
+    when_i_visit_the_homepage
+    and_i_submit_the_url_of_the_content_item
+
+    when_i_add_an_additional_tag
+    and_i_submit_the_form
+
+    then_just_that_link_is_sent_to_the_publishing_api
+  end
+
+  def given_we_can_populate_the_dropdowns_with_content_from_publishing_api
+    publishing_api_has_topics([
+      "/topic/id-of-already-tagged",
+      "/topic/business-tax/pension-scheme-administration",
+    ])
+  end
+
+  def given_there_is_an_item_that_can_have_only_one_link_type
+    publishing_api_has_lookups(
+      '/my-content-item' => 'MY-CONTENT-ID'
+    )
+
+    stub_request(:get, "#{PUBLISHING_API}/v2/content/MY-CONTENT-ID")
+      .to_return(body: {
+        # see blacklisted-tag-types.yml for this
+        publishing_app: "test-app-that-can-be-tagged-to-topics-only",
+        content_id: "MY-CONTENT-ID",
+        base_path: '/my-content-item',
+        format: 'mainstream_browse_page',
+        title: 'This Is A Content Item',
+      }.to_json)
+
+    stub_request(:get, "#{PUBLISHING_API}/v2/links/MY-CONTENT-ID")
+      .to_return(body: {
+        content_id: "MY-CONTENT-ID",
+        links: {
+          topics: "ID-OF-ALREADY-TAGGED",
+          mainstream_browse_pages: "ID-OF-ALREADY-TAGGED-BROWSE-PAGE",
+        },
+        version: 54_321,
+      }.to_json)
+  end
+
+  def when_i_visit_the_homepage
+    visit root_path
+  end
+
+  def and_i_submit_the_url_of_the_content_item
+    fill_in 'content_lookup_form_base_path', with: '/my-content-item'
+    click_on 'Show content item'
+  end
+
+  def when_i_add_an_additional_tag
+    @tagging_request = stub_request(:patch, "#{PUBLISHING_API}/v2/links/MY-CONTENT-ID")
```

Comment:
can we move this to `and_i_submit_the_form` below? bit confusing when reading through.